src/core/agent/services/multiSigService.test.ts

```diff
@@ -1406,6 +1407,84 @@ describe("Multisig sig service of agent", () => {
     expect(result.threshold).toBe(3);
   });
 
+  test("Should can be ready to rotate multisig", async () => {
+    Agent.agent.getKeriaOnlineStatus = jest.fn().mockReturnValueOnce(true);
+    const multisigIdentifier = "newMultisigIdentifierAid";
+    const signifyName = "newUuidHere";
+    identifiersGetMock = jest.fn().mockResolvedValue(aidReturnedBySignify);
+    identifiersCreateMock = jest.fn().mockResolvedValue({
+      identifier: multisigIdentifier,
+      signifyName,
+    });
+    identifierStorage.getIdentifierMetadata = jest
+      .fn()
+      .mockResolvedValue(keriMetadataRecord);
+    mockResolveOobi = jest.fn().mockResolvedValue({
+      name: "oobi.AM3es3rJ201QzbzYuclUipYzgzysegLeQsjRqykNrmwC",
+      metadata: {
+        oobi: "testOobi",
+      },
+      done: true,
+      error: null,
+      response: {},
+      alias: "c5dd639c-d875-4f9f-97e5-ed5c5fdbbeb1",
+    });
+
+    identifiersMemberMock = jest.fn().mockResolvedValue({
+      signing: [
```

Comment:
This test isn't correct. If it's a multi-sig, there should be multiple values returned here for signing and rotation.

Please add 2 signing members - one that is has rotated and the other that isn't. The final response should just show the one that has already rotated.